doc/sphinx/linux_install_guide/moduleonly.rst

```diff
@@ -41,6 +41,12 @@ Now you can configure. I strongly recommend building in a separate directory.
 Remember to add the argument *--with-biosig* to the configure script 
 if you want to have extra biomedical fileformats for stfio. 
 
+If you want to install the *stfio* module for a non-default Python distribution, such as **Anaconda Python**, use the argument *--prefix=* to specify the installation path as the *site-packages* folder of your favorite Python distribution. The following example will install *stfio* for Anaconda Python 2.7:
+
+::
+
+    $ ../../configure --enable-module --prefix=$HOME/anaconda/lib/python2.7/site-packages
```

Comment:
It worked for me without having to specify `PYTHON=`. In my case the default Python is already Anaconda: `which python` points to the `$HOME/anaconda/bin/python`. Do you think `PYTHON=` would be required for other cases?